Cargo.toml

```diff
@@ -27,7 +27,3 @@ std = [
   "schnorrkel",
 ]
 
-[workspace]
```

Comment:
js is now a standalone crate as workspace prevented configuring cargo release profile for JS only